### DISCUSSION TITLE
Welcome to Nightingale Discussions!

### DISCUSSION THREAD

**Discussion by dpaulino** [jenius-apps/nightingale-rest-api-client#150] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tbolon** [comment]
Hello @dpaulino!

First, thank you for this wonderful project.

I was looking for a way to share REST requests with my friends in our projects, and was desperate looking for a tool which could use local filesystem (so I could push requests sibling the API server code).

I love the polishing of the app! Great work.

**Reply by dpaulino** [reply]
Thanks!

**Comment by AshrafSada** [comment]
This is really a Jenius App, Thank you for the wonderful interface, although I have a small suggestion that is available on postman, which is the UI zooming or text enlargement.

I would really appreciate it if you add this feature to Nightingale, it will be a great feature.

Thank you
